scene/gui/graph_edit.cpp

```diff
@@ -399,7 +430,18 @@ void GraphEdit::add_child_notify(Node *p_child) {
 		gn->connect("selected", callable_mp(this, &GraphEdit::_graph_node_selected).bind(gn));
 		gn->connect("deselected", callable_mp(this, &GraphEdit::_graph_node_deselected).bind(gn));
 		gn->connect("slot_updated", callable_mp(this, &GraphEdit::_graph_node_slot_updated).bind(gn));
+
+		if (gn->is_comment()) {
+			background_nodes_separator_idx++;
```

Comment:
Do we actually need to track this? Can't we update connections layer's index along the comments? And in `_graph_node_raised` we can probably just refer to the connection layer's index.

I'm just concerned that adding a tracker like that would make the code too fragile.